build.sc

```diff
@@ -70,9 +70,9 @@ object Deps {
     val scalanativeTestRunner = ivy"org.scala-native::test-runner:0.4.0"
   }
 
-  val acyclic = ivy"com.lihaoyi::acyclic:0.2.0"
-  val ammonite = ivy"com.lihaoyi:::ammonite:2.4.1"
-  val ammoniteTerminal = ivy"com.lihaoyi::ammonite-terminal:2.4.1"
+  val acyclic = ivy"com.lihaoyi:::acyclic:0.3.0"
```

Comment:
I guess this would fail, once used, as acyclic is now cross-compiled against the full Scala version.

Reply:
We should remove the acyclic bump from this PR and handle it separately.

Reply:
See #1618